roles/splunk_common/tasks/install_splunk_tgz.yml

```diff
@@ -0,0 +1,14 @@
+- name: Install Splunk (Linux)
+  unarchive:
+    src: "{{ splunk.build_location }}"
+    dest: "{{ splunk.opt }}"
+    owner: "{{ splunk.user }}"
+    group: "{{ splunk.group }}"
+    remote_src: "{{ splunk.build_remote_src }}"
+  register: install_result
+  until: install_result is succeeded
+  retries: "{{ retry_num }}"
+  delay: 3
+  become: yes
+  become_user: "{{ privileged_user }}"
```

Comment:
Not sure if we should be installing as privileged user.

Reply:
I think this is probably necessary on most systems, although for the linux ones we'll want to switch users.  We'll want to verify that the appropriate user is set downstream